### PR TITLE
Adding couchers project

### PR DESCRIPTION
**Pull request by ShmuelTreiger**
We are looking for devs and already have tags for first issues. I think we're a great fit

### CODE REVIEW

File: _data/projects/couchers.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - python
+  - sqlalchemy
+  - grpc
+  - javascript
+  - reactjs
+  - typescript
+upforgrabs:
+  name: good first issue
+  link: https://github.com/Couchers-org/couchers/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/couchers.yml]:
```suggestion
  link: https://github.com/Couchers-org/couchers/labels/good%20first%20issue
```
That would be better.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@ShmuelTreiger Thanks!!

**Comment by ShmuelTreiger** [comment]
Thank you for maintaining this project!

**Comment by ritwik12** [comment]
> Thank you for maintaining this project!

Your welcome :)

**Pull request merged**